content/swift/concepts/optionals/optionals.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: 'Optionals'
+Description: 'Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Optionals'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Optionals are types that deal with scenarios involving the potential presence or absence of a value (`nil`).
+
+## Syntax
+
+In Swift, a non-optional variable should not be set to `nil`. Rather, the optional type must be defined using the added question mark `?`.
+
+There are two ways to use an optional. Below, a variable named `myVariable` is set to an optional `Type`, `Type?`. `Type` refered to a [Swift data type](https://www.codecademy.com/resources/docs/swift/data-types). It can either be equal to some value with the specified data type or `nil`.
```

Comment:
```suggestion
There are two ways to use an optional. Below, a variable named `myVariable` is set to an optional `Type`, `Type?`. `Type` referred to a [Swift data type](https://www.codecademy.com/resources/docs/swift/data-types). It can either be equal to some value with the specified data type or `nil`.
```